pkg/skaffold/docker/client.go

```diff
@@ -221,7 +206,7 @@ func getMinikubeDockerEnv(minikubeProfile string) (map[string]string, error) {
 		env[kv[0]] = kv[1]
 	}
 
-	if found, _ := detectWsl(); found {
+	if found, _ := util.DetectWSL(); found {
 		cmd := exec.Command("wslpath", env["DOCKER_CERT_PATH"])
```

Comment:
I know this isn't part of your change but could you just add a comment here to the effect of:
```
// rewrite Unix path to Windows
```

(at least, I'm assuming Minikube in WSL reports a unix-style path?)

Reply:
done